examples/views/wait_for_button_press.py

```diff
@@ -0,0 +1,53 @@
+import asyncio
+
+import discord
+from discord.ext import commands
+from discord.ui import Button, View
+
+
+bot = commands.Bot(command_prefix="!")
+
+# Create a slash command for the supplied guilds.
+@bot.slash_command(name="button", guild_ids=[...], description="Wait for button press example from pycord") 
+async def button(ctx):
+    ###############
+    # Create button and add it to View()
+    button = Button(style=discord.ButtonStyle.green, label="Click me!")
+    view = View()
+    view.add_item(button)
+    ###############
+    
+    ###############
+    # Create embed and send message with button and embed.
+    embed = discord.Embed(description=f'Wait for button press example')
+    embed.set_author(name=f'Pycord-Development')
```

Comment:
F-strings are not needed here. Also switch the single quotes (`'`) to double quotes (`"`)